internal/services/databricks/databricks_workspace_resource_test.go

```diff
@@ -780,7 +780,11 @@ resource "azurerm_databricks_workspace" "test" {
 func (DatabricksWorkspaceResource) machineLearning(data acceptance.TestData, sku string) string {
 	return fmt.Sprintf(`
 provider "azurerm" {
-  features {}
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false
+    }
+  }
```

Comment:
how come this is needed here? we should look into the root cause of this and try to fix it so we don't need an exception as users will experience this as well

Reply:
@katbyte, This is due to policy automatically creating NSG's if the LRO takes too long, it's hit or miss.